frontend/pages/admin/IntegrationsPage/IntegrationsPage.tsx

```diff
@@ -8,10 +8,14 @@ import getFilteredIntegrationSettingsNavItems from "./IntegrationNavItems";
 const baseClass = "integrations";
 
 interface IIntegrationSettingsPageProps {
+  router: any;
```

Comment:
Will the `InjectedRouter` type work for this?

Reply:
yep will add this.